modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java

```diff
@@ -491,8 +497,8 @@ private String getSessionDataKeyConsent(CloseableHttpClient client, String sessi
      * @param idToken  Encrypted ID token to be decrypted and checked.
      * @param audience Audience value that should be appeared in the token.
      * @return Boolean True if audience matches, False otherwise.
-     * @throws ParseException
-     * @throws JOSEException
+     * @throws ParseException Error
```

Comment:
Missing fullstops at the end

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096